solidity/ecdsa/deploy/90_upgrade_random_beacon_chaosnet.ts

```diff
@@ -0,0 +1,30 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { getNamedAccounts, deployments } = hre
+  const { governance } = await getNamedAccounts()
+  const { execute } = deployments
+
+  const RandomBeaconChaosnet = await deployments.get("RandomBeaconChaosnet")
+
+  // Upgrade the random beacon smart contract in `WalletRegistry` to
+  // `RandomBeaconChaosnet`. This is a temporary solution to enable usage of
+  // `WalletRegistry` before the random beacon functionalities in the client
+  // are ready.
+  await execute(
+    "WalletRegistryGovernance",
+    { from: governance, log: true, waitConfirmations: 1 },
```

Comment:
I do not think this will work for Goerli. The governance address is an external address, owned by the dev team. I think we should move this step earlier, before `WalletRegistryTransferGovernance`.

Reply:
In a7859770d0342956f7049a3a468fe74c181c3695 I moved upgrading `RandomBeaconChaosnet` just after `WalletRegistry` deployment.